README.md

```diff
@@ -1,17 +1,39 @@
-# SOEN341-InsertName
+﻿# SOEN341-InsertName
 ### Authors
-* Simon Lim (tech lead) | 
-* Émilie Martin | @ emilie-martin
-* Daniela Venuta | @ daniela-venuta
-* Minh-Tam Do |
-* Luc Nguyen |
-* Karin Kazarian |
+* Simon Lim (tech lead) | @nasaku898
+* Émilie Martin | @emilie-martin
+* Daniela Venuta | @daniela-venuta
+* Minh-Tam Do | @mint47
+* Luc Nguyen | @1999lucnguyen
+* Karin Kazarian | @Karin-kazarian
+
+### Description
+This is a project partial fulfillement of the requirements for the course SOEN341 at Concordia University.
+### Objectives
+The app is a replication of the social media Instagram from which we will add on features.
```

Comment:
For the objectives, I would write something more along the lines of "The app is intended to implement the core features of Instagram, such that the user is able to post a picture, follow another user, and be allowed to comment under posted pictures." because that's what we want our app to do (à la base) and we can always come back and add our additional features after having decided what they will be.